tailwind/plugins/base/gc-base.js

```diff
@@ -6,7 +6,6 @@ module.exports = function () {
   return function ({ addBase, config }) {
     addBase({
       html: {
-        overflowY: 'visible !important', // Need this for sticky nav to work
```

Comment:
This was for Gatsby and for Travel Study - we don't need it here. It interferes with how the body scroll locking work for the Headless UI mobile.